### PR TITLE
feat: Set lang and auth filter in password-reset

### PR DESCRIPTION
**Pull request by bodnara**
* password-reset app now reacts to language and auth filter in url

### CODE REVIEW

File: apps/password-reset/src/app/services/password-reset-config.service.ts [hunk 1]
```diff
@@ -14,15 +14,29 @@ export class PasswordResetConfigService {
...
+    private authzSevice: AuthzResolverService
   ) { }
 
   loadConfigs(): Promise<void> {
     return this.appConfigService.loadAppDefaultConfig()
       .then(() => this.appConfigService.loadAppInstanceConfig())
       .then(() => this.setApiUrl())
-      .then(() => this.appConfigService.setInstanceFavicon());
+      .then(() => this.appConfigService.setInstanceFavicon())
+      .then(() => this.initAuthService.verifyAuth())
```
Comment (xkureck) [inline apps/password-reset/src/app/services/password-reset-config.service.ts:25]: the authentication should happen only and only if in the url ISNT "token" parameter

File: apps/password-reset/src/app/app.component.ts [hunk 2]
```diff
@@ -13,18 +15,22 @@ import { TokenExpiredDialogComponent } from './dialogs/token-expired-dialog/toke
...
+    let mode = '';
     if (queryParams.includes('activation')) {
-      mode = 'activation'
+      mode = 'activation';
     } else if(queryParams.includes('token')) {
-      mode = 'reset'
+      mode = 'reset';
     } else {
-      mode = 'change'
+      mode = 'change';
```
Comment (xkureck) [inline apps/password-reset/src/app/app.component.ts:33]: we disscuss with pavel that mode change will not be in password reser app
please remove it

**Review by xkureck — CHANGES_REQUESTED** [review]

File: apps/password-reset/src/app/app.component.ts (hunk 2 above)
Comment (bodnara) [reply apps/password-reset/src/app/app.component.ts]: I don't think this should be part of this PR as this was about setting language and auth filter from query params.

File: apps/password-reset/src/app/services/password-reset-config.service.ts [hunk 3]
```diff
@@ -14,15 +14,36 @@ export class PasswordResetConfigService {
...
   loadConfigs(): Promise<void> {
     return this.appConfigService.loadAppDefaultConfig()
       .then(() => this.appConfigService.loadAppInstanceConfig())
       .then(() => this.setApiUrl())
-      .then(() => this.appConfigService.setInstanceFavicon());
+      .then(() => this.appConfigService.setInstanceFavicon())
+      .then(() => {
+        const queryParams = location.search.substr(1);
+        if(!queryParams.includes('token')) {
+          return this.initAuthService.verifyAuth();
```
Comment (xkureck) [inline apps/password-reset/src/app/services/password-reset-config.service.ts:28]: the authentication is not working at all, I think the problem is in the thing that you dont wait for the result, there is not a promise in this {}

**Review by xkureck — CHANGES_REQUESTED** [review]

File: apps/password-reset/src/app/services/password-reset-config.service.ts (hunk 3 above)
Comment (xkureck) [reply apps/password-reset/src/app/services/password-reset-config.service.ts:28]: and rebase please

**Review by xkureck — APPROVED** [review]

**Pull request merged**